streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapterTest.java

```diff
@@ -306,4 +327,33 @@ public String name() {
 
         return parameters;
     }
+
+    @Test
+    public void shouldLogWarningWhenSettingWalOptions() throws Exception {
+
+        try (final LogCaptureAppender appender = LogCaptureAppender.createAndRegister(RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter.class)) {
+
+            final RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter adapter
+                    = new RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter(new DBOptions(), new ColumnFamilyOptions());
+
+            for (final Method method : RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter.class.getDeclaredMethods()) {
+                if (walRelatedMethods.contains(method.getName())) {
+                    method.invoke(adapter, getDBOptionsParameters(method.getParameterTypes()));
+                }
+            }
+
+            final List<String> walOptions = Arrays.asList("walDir", "walFilter", "walRecoveryMode", "walBytesPerSync", "walSizeLimitMB", "manualWalFlush", "maxTotalWalSize", "walTtlSeconds");
```

Comment:
nit: Would it be possible to extract this options from `walRelatedMethods`?

Reply:
@cadonna 
I Thought about it also.
To do that I need to remove the "set" from the method name and to de-capitalize the first letter of the option. 
I think this is a bit overkill because:
1. I got checkstyle errors trying to use something like toLowerCase()... 
2. Its going to be ugly to do it without additional StringUtils helper libs (like apache commons-lang StringUtils which I saw we dont have)

Reply:
Fair enough!